src/pm/agent/session.py

```
@@ -0,0 +1,311 @@
+            permissions = project_config.agent_config.permissions
+
+        launch_opts = LaunchOptions(
+            prompt=session.task_description or "",
+            work_dir=session.worktree_path or "",
+            model=model,
+            permissions=permissions,
+        )
+        agent_cmd = agent.launch_command(launch_opts)
+        cmd_str = " ".join(agent_cmd)
+
+        subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)